spec/numberformat.html

```diff
@@ -30,6 +30,8 @@ <h1>SetNumberFormatDigitOptions ( _intlObj_, _options_, _mnfdDefault_, _mxfdDefa
           1. Set _intlObj_.[[MaximumSignificantDigits]] to _mxsd_.
         1. Else if _mnfd_ is not *undefined* or _mxfd_ is not *undefined*, then
           1. Set _intlObj_.[[RoundingType]] to ~fractionDigits~.
+          1. If _mxfd_ is not *undefined*, then
+            1. Set _mnfdDefault_ to min( _mxfd_, _mnfdDefault_ ).
```

Comment:
Can you make a variable `_mnfdActualDefault_` since we already have `_mxfdActualDefault_`?

Reply:
I did it that way at the beginning, but chose not to at end for brevity (it would have required a two branched condition instead). Will do.